test/lib/factories/EnginePluginsFactory.test.ts

```diff
@@ -7,6 +7,6 @@ describe('EnginePluginsFactoryImpl', () => {
 		const enginePlugins = pluginsFactory.create();
 
 		expect(enginePlugins).toHaveLength(1);
-		expect(enginePlugins[0].getAvailableEngineNames()).toEqual(['SampleEngine1', 'SampleEngine2']);
+		expect(enginePlugins[0].getAvailableEngineNames()).toEqual(['SampleEngine1', 'SampleEngine2', 'SampleEngine3']);
```

Comment:
Currently, the tests for the EnginePluginFactory still use the temporary engines. Once we move to the actual engines, this test will change anyway, so for now I'm just keeping it up-to-date with the additional temporary engine.